docs/editors/overview.md

```diff
@@ -345,7 +338,6 @@ extensions, see [integrating a new editor](new-editor.md).
 
 Metals does not support the following features:
 
-- Rename symbol
```

Comment:
I think we should remove this only once the stable release is out - I have a number of updates that will go together with that.

Reply:
Sounds good. I'll add it back in.